Jenkinsfile

```diff
@@ -0,0 +1,614 @@
+#!groovy
+
+def initializeEnvironment() {
+  env.DRIVER_DISPLAY_NAME = 'CassandraⓇ Java Driver'
+  env.DRIVER_METRIC_TYPE = 'oss'
+  if (env.GIT_URL.contains('riptano/java-driver')) {
+    env.DRIVER_DISPLAY_NAME = 'private ' + env.DRIVER_DISPLAY_NAME
+    env.DRIVER_METRIC_TYPE = 'oss-private'
+  } else if (env.GIT_URL.contains('java-dse-driver')) {
+    env.DRIVER_DISPLAY_NAME = 'DSE Java Driver'
+    env.DRIVER_METRIC_TYPE = 'dse'
+  }
+
+  env.GIT_SHA = "${env.GIT_COMMIT.take(7)}"
+  env.GITHUB_PROJECT_URL = "https://${GIT_URL.replaceFirst(/(git@|http:\/\/|https:\/\/)/, '').replace(':', '/').replace('.git', '')}"
+  env.GITHUB_BRANCH_URL = "${GITHUB_PROJECT_URL}/tree/${env.BRANCH_NAME}"
+  env.GITHUB_COMMIT_URL = "${GITHUB_PROJECT_URL}/commit/${env.GIT_COMMIT}"
+
+  env.MAVEN_HOME = "${env.HOME}/.mvn/apache-maven-3.3.9"
+  env.PATH = "${env.MAVEN_HOME}/bin:${env.PATH}"
+  env.JAVA_HOME = sh(label: 'Get JAVA_HOME',script: '''#!/bin/bash -lex
```

Comment:
Fero said something about removing the -x switch to reduce verbosity, is that planned for later?

Reply:
We can, but the general approach has been that most of the time, you don't even need to look at this stuff. But when you do, it's usually because something is not working as expected, and then it's helpful to have more output (even if it's a large mass to sift through).